Source/WebCore/dom/ScriptType.h

```diff
@@ -28,8 +28,8 @@
 namespace WebCore {
 
 // https://html.spec.whatwg.org/multipage/scripting.html#concept-script-type
-enum class ScriptType : uint8_t { Classic, Module, ImportMap };
-static constexpr unsigned bitWidthOfScriptType = 2;
-static_assert(static_cast<unsigned>(ScriptType::ImportMap) <= ((1U << bitWidthOfScriptType) - 1));
+enum class ScriptType : uint8_t { Classic, Module, ImportMap, SpeculationRules };
+static constexpr unsigned bitWidthOfScriptType = 3;
```

Comment:
There are now 4 values.  Shouldn't those still fit in 2 bits?

Reply:
Oops, yeah

Reply:
Oops, yeah